src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -36,6 +47,21 @@ public void SendAsync_SimpleGet_Success()
         }
 
         [Fact]
+        public void SendAsync_SimpleHttpsGet_Success()
+        {
+            var handler = new HttpClientHandler();
+            var client = new HttpClient(handler);
+
+            var response = client.GetAsync(HttpTestServers.SecureRemoteGetServer).Result;
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            var responseContent = response.Content.ReadAsStringAsync().Result;
+            Assert.Equal<string>(response.ReasonPhrase, "OK");
+            _output.WriteLine(responseContent);
+            handler.Dispose();
+            client.Dispose();
```

Comment:
Since the expected use case is to await the async operations rather than block synchronously for them, I'd suggest these tests be written as "async Task" rather than "void", e.g.
```
[Fact]
public async Task SendAsync_SimpleHttpsGet_Success()
{
    using (var client = new HttpClient())
    {
        HttpResponseMessage response = await client.GetAsync(HttpTestServers.SecureRemoteGetServer);
        Assert.Equal(HttpStatusCode.OK, response.StatusCode);
        Assert.Equal("OK", response.ReasonPhrase);

        string responseContent = await response.Content.ReadAsStringAsync();
        _output.WriteLine(responseContent);
    }
}
```
A few other things to note there:
- The expected value should come first in Assert.Equal; you did that for the first one, but the second one had its arguments reverse
- I've used a using to cleanup deterministically so that if an assert does fail, we don't rely on the finalizer to clean up and potentially impact other tests more than necessary
- I've avoided the use of var except in cases where the type is easily clear from the right-hand side, e.g. we try to only use var in the cast of constructing an object, a cast, etc.

Separately, is there anything we can verify about responseContent?